src/locales/de.json

```
@@ -351,6 +351,7 @@
         "Jobs": "Drucke",
         "LastModified": "Zuletzt geändert",
         "LayerHeight": "Schichthöhe",
+        "LoadCompleteHistory": "Lade komplette Verlaufsliste",
```

We use `Historie` on different keys why the switch to `Verlaufsliste`?

```suggestion
        "LoadCompleteHistory": "Lade vollständige Historie",
```